src/Integration.Vsix/SonarLintTagger/TaggerProvider.cs

```diff
@@ -50,23 +49,24 @@ internal sealed class TaggerProvider : IViewTaggerProvider, ITableDataSource, II
         internal readonly ITextDocumentFactoryService TextDocumentFactoryService;
         internal readonly IContentTypeRegistryService ContentTypeRegistryService;
         internal readonly IFileExtensionRegistryService FileExtensionRegistryService;
-        internal readonly _DTE dte;
+        internal readonly IServiceProvider serviceProvider;
 
         private readonly List<SinkManager> managers = new List<SinkManager>();
         private readonly TrackerManager taggers = new TrackerManager();
 
         private readonly ISonarLintDaemon daemon;
         private readonly ISonarLintSettings settings;
-
+        private readonly ISonarLintOutput logger;
 
         [ImportingConstructor]
         internal TaggerProvider(ITableManagerProvider provider,
             ITextDocumentFactoryService textDocumentFactoryService,
             IContentTypeRegistryService contentTypeRegistryService,
             IFileExtensionRegistryService fileExtensionRegistryService,
             ISonarLintDaemon daemon,
-            SVsServiceProvider serviceProvider,
-            ISonarLintSettings settings)
+            [Import(typeof(SVsServiceProvider))] IServiceProvider serviceProvider,
```

Comment:
Fixed the method signature to match the normal pattern for importing the service provider in VS.